perception/image_projection_based_fusion/src/roi_cluster_fusion/node.cpp

```diff
@@ -169,7 +169,7 @@ void RoiClusterFusionNode::fuseOnSingleImage(
   }
 
   for (const auto & feature_obj : input_roi_msg.feature_objects) {
-    int index = 0;
+    int index = -1;
```

Comment:
How about the following?

```suggestion

    int index = 0;
    bool associated = false;
    double max_iou = 0.0;
    for (const auto & cluster_map : m_cluster_roi) {
      double iou(0.0), iou_x(0.0), iou_y(0.0);
@ -191,6 +191,10 @@ void RoiClusterFusionNode::fuseOnSingleImage(
        max_iou = iou + iou_x + iou_y;
      }
    }
    if (!associated) {
      continue;
    }

```

Reply:
@yukkysaito Thanks for suggestion. It applied in [e83de7a](https://github.com/autowarefoundation/autoware.universe/pull/4445/commits/e83de7aff03217cc347fff296c1523580d27d21a).